### PR TITLE
BASIC sound

### PR DESCRIPTION
**Pull request by mooinglemur**
https://github.com/commanderx16/x16-rom/pull/378

### CODE REVIEW

File: basic/token2.s [hunk 1]
```diff
@@ -187,6 +214,22 @@ brktxt	.byt $d
 err30	.byt "BREAK",0,$a0 ;shifted space
 err31	.byt "NOT GRAPHICS MOD", 'E'+$80
 errngm	=31
+err32	.byt "INVALID CHANNE", 'L'+$80
```
Comment (indigodarkwolf) [inline basic/token2.s:217]: Yeah, my only feedback is that if we're tight on bytes we could replace "INVALID" with "BAD"

**Review by indigodarkwolf — APPROVED** [review]
👍

File: basic/token2.s (hunk 1 above)
Comment (jestin) [reply basic/token2.s]: Removed additional words altogether ;)

Comment (indigodarkwolf) [reply basic/token2.s]: Yes! Removing the superfluous adjectives altogether! Save the bytes! <3

**Pull request merged**